### ISSUE TITLE
less() and edit() are not defined for macros

### ISSUE THREAD

**Issue by diegozea** [JuliaLang/julia#3377]
```
julia> less( @b_str )
ERROR: wrong number of arguments

julia> less( b_str )
ERROR: b_str not defined

julia> less
# methods for generic function less
less(file::String,line::Integer) at util.jl:174
less(file::String) at util.jl:177
less(f::Function) at util.jl:181
less(f::Function,t) at util.jl:182
```

**Comment by dhoegh** [comment]
Bump somehow this did not get auto closed when  #15282 was merged.

**Comment by yuyichao** [comment]
.... not even the PR was closed when I merge it with the merge butten .... https://github.com/JuliaLang/julia/pull/15282#issuecomment-205982982
